.gitignore

```diff
@@ -10,9 +10,6 @@ bin
 # Test binary, build with `go test -c`
 *.test
 
-# Output of the go coverage tool, specifically when used with LiteIDE
-*.out
-
```

Comment:
Unrelated (and potentially wrong?) change

Reply:
The `verify-deps` job was [previously complaining about this file](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_cluster-baremetal-operator/466/pull-ci-openshift-cluster-baremetal-operator-main-verify-deps/1923328959299194880).

Reply:
Oh lovely... Can we bump go-errors to include https://github.com/go-errors/errors/commit/7db060dd91303e294f1ee2855024347369f42c11?

Reply:
Done